### ISSUE TITLE
Multi-class output for binary classification

### ISSUE THREAD

**Issue by meechos** [hussius/tabnet_fork#5]
The original tabnet classifier by google is hard-coded to pass predictions in a multi-class format, regardless of whether `num_classes` is 2.

Would you know if the above means
- there are two output neurons in the model
- performance is affected for binary classification problems?

Is your implementation similar in this aspect?

Many thanks for a great repository!

**Comment by hussius** [comment]
Hi! I haven't changed that part of the implementation - indeed, I didn't think about it until now.

I would assume that there would be two output neurons, yes. The performance should not be affected (based on past experience with Keras models, where I have tried one output neuron with sigmoid vs two output neurons with categorical cross-entropy).

By the way, I have learned there is a nice and more flexible Pytorch implementation of TabNet: 
https://github.com/dreamquark-ai/tabnet

My repo is basically just adding a CLI to the original Tensorflow implementation.
